docs/documentation/development/bindings/faq.md

```diff
@@ -21,18 +21,18 @@ Here is a list of frequently asked question around the development of bindings.
     
     If your system does not provide you any possibility to get hold of such information, but rather only shows you a "logical" view of it, then 1) is also a valid option to pursue.
   
-  1. _Do I have to create XML files in `ESH-INF/thing` for all devices or is there any other option?_
+2. _Do I have to create XML files in `ESH-INF/thing` for all devices or is there any other option?_
```

Comment:
IIRC multiple usage of "1." is valid for an enumeration and will be replaced by the correct number. Isn't that valid for all Markdown implementations?